docs/schemas/profile/rules.json

```diff
@@ -0,0 +1,209 @@
+                "My Rule Name"
+            ],
+            "title": "The Name Schema",
+            "type": "string"
+        }
+    },
+    "definitions": {
+        "conditionArray": {
+            "type": "array",
+            "title": "Condition Array",
+            "description": "An array of conditions with a possible recursive inclusion of another condition array.",
+            "default": [],
+            "items": {
+                "anyOf": [
+                    {
+                        "$ref": "#/definitions/conditions"
+                    },
+                    {
+                        "$ref": "#/definitions/condition"
+                    }
+                ]
+            }
+        },
+        "condition": {
+            "type": "object",
+            "title": "Condition",
+            "description": "Rule conditions are a combination of facts, operators, and values that determine whether the rule is a success or a failure. The simplest form of a condition consists of a fact, an operator, and a value. When the engine runs, the operator is used to compare the fact against the value. Sometimes facts require additional input to perform calculations. For this, the params property is passed as an argument to the fact handler. params essentially functions as fact arguments, enabling fact handlers to be more generic and reusable.",
+            "default": {
+                "fact": "my-fact",
+                "operator": "lessThanInclusive",
+                "value": 1
+            },
+            "examples": [
+                {
+                    "fact": "gameDuration",
+                    "operator": "equal",
+                    "value": 40.0
+                },
+                {
+                    "value": 5.0,
+                    "fact": "personalFoulCount",
+                    "operator": "greaterThanInclusive"
+                },
+                {
+                    "fact": "product-price",
+                    "operator": "greaterThan",
+                    "path": "$.price",
+                    "value": 100.0,
+                    "params": {
+                        "productId": "widget"
+                    }
+                }
+            ],
+            "required": [
+                "path",
+                "operator",
+                "value"
+            ],
+            "properties": {
+                "operator": {
+                    "type": "string",
+                    "anyOf": [
+                        {
+                            "const": "equal",
+                            "title": "fact must equal value"
+                        },
+                        {
+                            "const": "notEqual",
+                            "title": "fact must not equal value"
+                        },
+                        {
+                            "const": "these",
+                            "title": "erators use strict equality (===) and inequality (!==)"
```

Comment:
What do you mean by erators ?

Reply:
this is operator and it must be in description of `equal` and `notEqual`

Reply:
Solved. Thanks!